lambdas/get-addresses/src/app.ts

```diff
@@ -1,11 +1,17 @@
 import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
 import { AddressService } from "./services/address-service";
-import config from "./config";
 import { DynamoDbClient } from "./lib/dynamo-db-client";
+import { SsmClient } from "./lib/param-store-client";
+import { ConfigService } from "./services/config-service";
+
+const configService = new ConfigService(SsmClient);
+const initPromise = configService.init();
```

Comment:
initialise the config service promise outside the lambdaHandler. This will allow Lambda to cache the ssm results.

However we would need to resolve it inside the handler. (very painful to test)